site/zenodo_rdm/migrator/stream.py

```diff
@@ -0,0 +1,17 @@
+
+from invenio_rdm_migrator.streams import Stream
+from invenio_rdm_migrator.streams.records.load import RDMRecordCopyLoad
+
+from extract import JSONLExtract
+from transform import ZenodoToRDMRecordTransform
+
+
+class RecordStream(Stream):
+    """ETL stream for Zenodo to RDM records."""
+
+    def __init__(self, filename, db_uri, output_path):
+        """Constructor."""
+        super().__init__(
+            extract=JSONLExtract(filename),
+            transform=ZenodoToRDMRecordTransform(),
+            load=RDMRecordCopyLoad(db_uri, output_path),
```

Comment:
IMO it is input_path since its the input to load from no?

Reply:
As any other question in the migration sprint, the answer is "yes and no". While you are correct on the fact that we are reading/loading data from that directory, namely the csv files to `COPY`; we are also writing/dumping files there, namely those csv files produced by the `prepare` step.

So yes, we are loading, but we are also dumping. What about `tmp_dir` or `data_dir` (dir or path)?

Reply:
Fixed, discussed IRL it's `tmp_dir`